dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ResourcesServiceImpl.java

```diff
@@ -28,6 +26,7 @@
 
 import org.apache.dolphinscheduler.api.constants.ApiFuncIdentificationConstant;
 import org.apache.dolphinscheduler.api.dto.resources.ResourceComponent;
+import org.apache.dolphinscheduler.api.dto.resources.ResourceContent;
```

Comment:
Please split this part into other PR.

Reply:
do you means  we should not to modify `ResourcesServiceImpl` ?

Reply:
We should split it according to the comment https://github.com/apache/dolphinscheduler/pull/10998#issuecomment-1207708303

Reply:
Yes , I understand 。  I've made the smallest and necessary change in ResourcesServiceImpl。 If we don’t do this， it will be weird when convert to DTO。 I'm not sure about this boundary  according to  comment https://github.com/apache/dolphinscheduler/pull/10998#issuecomment-1207708303

Reply:
@SbloodyS  I have split  ，please     approve